pkg/kubelet/eviction/eviction_manager.go

```diff
@@ -62,6 +63,8 @@ type managerImpl struct {
 	resourceToRankFunc map[api.ResourceName]rankFunc
 	// resourceToNodeReclaimFuncs maps a resource to an ordered list of functions that know how to reclaim that resource.
 	resourceToNodeReclaimFuncs map[api.ResourceName]nodeReclaimFuncs
+	// machine info regarding the node
+	machineInfo *cadvisorapi.MachineInfo
```

Comment:
hmm... i would think to be consistent with `FsStats` we would add `Capacity` to `MemoryStats`, is there an objection to adding it?